source/javascripts/components/_Step.js

```diff
@@ -183,6 +169,10 @@ import { normalizeIconUrl } from "./StepItem/StepItem";
 			return !!this.libraryURL;
 		};
 
+		Step.prototype.isVCSStep = function() {
```

Comment:
What does `VCS` stand for? 🤔

Reply:
version control system, I've heard it as "official" umbrella term for git, TFS, SVN etc